File: Maximum in Struct Array/MaximumInArray.cpp
#include <bits/stdc++.h>
using namespace std;
struct Height {
	int feet;
	int inches;
};
int findMax(Height arr[], int n);
// driver program
int main()
{
	int t;
	cin>>t;
	while(t--)
	{
	    int n, tmp1, tmp2;
	    cin>>n;
	    Height arr[n];
	    for(int i=0; i<n; i++)
	    {
	        cin>>tmp1>>tmp2;
	        arr[i]={tmp1, tmp2};
	    }
	    int res = findMax(arr, n);
	    cout<<res<<endl;
	}
	return 0;
}
int findMax(Height arr[], int n)
{
    int maxValue = 0;
    for(int i = 0; i < n; i++){
        int height = arr[i].feet * 12 + arr[i].inches;
        if(height > maxValue){
            maxValue = height;
        }
    }
    return maxValue;
}

File: Maximum in Struct Array/MaximumInArray.java
import java.util.Scanner;

class Height {
    int feet;
    int inches;

    Height(int ft, int inc) {
        feet = ft;
        inches = inc;
    }
}

class MaximumInArray {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            int n = sc.nextInt();
            Height[] arr = new Height[n];
            for (int i = 0; i < n; i++) {
                int temp1 = sc.nextInt();
                int temp2 = sc.nextInt();
                arr[i] = new Height(temp1, temp2);

            }
            GfG gfg = new GfG();

            int res = gfg.findMax(arr, n);
            System.out.println(res);
        }
    }
}

class GfG {
    static int findMax(Height []arr, int n) {
        int max = 0;
        for (Height h : arr) {
            int height = h.feet * 12 + h.inches;
            if (height > max) {
                max = height;
            }
        }
        return max;
    }
}